### ISSUE TITLE
[UI] 'NoneType' object has no attribute 'group'

### ISSUE THREAD

**Issue by malloryfreeberg** [HumanCellAtlas/ingest-central#67]
**Give the bug a title**

[UI] 'NoneType' object has no attribute 'group'

**To Reproduce**

1. Go to staging UI
2. Upload spreadsheet
3. See error

**Expected behaviour**

* I expected the spreadsheet to upload. It has in staging before. The spreadsheet doesn't include a Schemas tab.

**Actual behaviour**

* Saw error

**Impact**

Cannot test pilot data spreadsheet upload/export.

**Screenshots or Messages**

![screen shot 2018-07-17 at 09 49 59](https://user-images.githubusercontent.com/10993501/42806644-cea9f40a-89a6-11e8-8693-78a2231615e1.png)

**Comment by pnejad** [comment]
I'm also experiencing this issue.

I was testing a spreadsheet on staging. This same spreadsheet had previously passed validation on staging.

**Comment by malloryfreeberg** [comment]
@justincc I have not seen this issue recently. Was there a diagnosis of the problem? Was there a solution and/or fix tested in dev?

**Comment by justincc** [comment]
Yes @aaclan-ebi and @rolando-ebi think this is fixed. We just wanted to check with you.

**Comment by aaclan-ebi** [comment]
Just to add, I believe this error was caused by wrong parsing of schema urls when system/ dir was added, this change fixed that issue long time ago https://github.com/HumanCellAtlas/ingest-client/commit/7a849db1eca1750efb5346ef0663f24af82f7bc6
